ouroboros-network/src/Ouroboros/Network/PeerSelection/RootPeersDNS/DNS/LedgerPeers.hs

```diff
@@ -69,7 +77,10 @@ newtype LedgerPeersConsensusInterface m = LedgerPeersConsensusInterface {
 
 -- | Trace LedgerPeers events.
 data TraceLedgerPeers =
-      PickedPeer RelayAccessPoint AccPoolStake PoolStake
+      TraceLedgerPeersDomains [DomainAccessPoint]
+    | TraceLedgerPeersResult  DNS.Domain [(IP, DNS.TTL)]
+    | TraceLedgerPeersFailure DNS.Domain DNS.DNSError
+    | PickedPeer RelayAccessPoint AccPoolStake PoolStake
       -- ^ Trace for a peer picked with accumulated and relative stake of its pool.
     | PickedPeers NumberOfPeers [RelayAccessPoint]
```

Comment:
For the sake of consistency, should we change the constructors of the other two? Or should we leave `Traceledger` from the other ones?